best-practices.md

```diff
@@ -203,6 +222,81 @@ register the media type with IANA, so that other STAC clients can find it easily
 it. It is relatively easy to [register](https://www.iana.org/form/media-types) a `vnd` media type.
 
+## Asset Roles
+
+[Asset roles](item-spec/item-spec.md#asset-roles) are used to describe what each asset is used for. They are particular useful 
+when several assets have the same media type, such as when an Item has a multispectral analytic asset, a 3-band full resolution 
+visual asset, a down-sampled preview asset, and a cloud mask asset, all stored as Cloud Optimized GeoTIFF (COG) images. It is 
+recommended to use at least one role for every asset available, and using multiple roles often makes sense. For example you'd use
+both `data` and `reflectance` if your main data asset is processed to reflectance, or `metadata` and `cloud` for an asset that 
+is a cloud mask, since a mask is considered a form of metadata (it's information about the data). Or if a single asset represents
+several types of 'unusable data' it might include `metadata`, `cloud`, `cloud-shadow` and `snow-ice`. If there is not a clear
+role in the [Asset Role Types](item-spec/item-spec.md#asset-role-types) or the following list then just pick a sensible name for 
+the role. And you are encouraged to add it to the list below and/or in an extension if you think the new role will have broader 
+applicability. 
+
+### List of Asset Roles
+
+In addition to the thumbnail, data and overview [roles listed](item-spec/item-spec.md#asset-role-types) in the Item spec, there
+are a number of roles that are emerging in practice, but don't have enough widespread use to justify standardizing them. So if
+you want to re-use other roles then try to find them on the list below, and also feel free to suggest more to include here.
+
+The 'source' field lists where the role comes from. The ones the say Item Spec are the only 'official' roles that are fully
+standardized. In time others on this list may migrate to a more 'official' list. Those that say 'best practice' are just from this 
+doc, the listing is the table below. The ones from extensions are mostly just 'best practices' in the extensions, as there are few
+actual role requirements.
+
+| Role Name | Source | Description                                                            |
+| --------- | -------------|----------------------------------------------------------------------- |
+| thumbnail | [Item Spec](item-spec/item-spec.md#asset-role-types) | An asset that represents a thumbnail of the item, typically a true color image (for items with assets in the visible wavelengths), lower-resolution (typically smaller 600x600 pixels), and typically a JPEG or PNG (suitable for display in a web browser). Multiple assets may have this purpose, but it recommended that the `type` and `roles` be unique tuples. For example, Sentinel-2 L2A provides thumbnail images in both JPEG and JPEG2000 formats, and would be distinguished by their media types. |
```

Comment:
Let's not repeat all roles from the extensions here. Better Add a list that links to additional roles specified in extensions.

Reply:
I think it is useful to at least get a sense of what all the potential roles are in one list. But I can play with listing them quickly here but not having all the detail.

Reply:
Maybe just a list?

* [SAR](extensions/sar/) (noise-power, amplitude, magnitude, ... [list all])
* EO...

Reply:
I added them to the table, but just one line for each with a list.